lib/utils.pm

```diff
@@ -739,6 +739,7 @@ test user as well as root.
 =cut
 sub ensure_serialdev_permissions {
     my ($self) = @_;
+    return if get_var('ROOTONLY');
```

Comment:
Please, describe variable in `variables.md`. Also I would introduce this check outside of this method, so it's not called in case we don't need these steps instead of triggering it everywhere.

Reply:
Calling it outside would mean needing to add it at 17 places and invite for the next person to forget to add it when calling `ensure_serialdev_permissions` so I do not want to do that.

Reply:
documented the variable.